### DISCUSSION TITLE
Error: invalid values

### DISCUSSION THREAD

**Discussion by avenbreaks** [0xPolygon/polygon-edge#768] [General] [answered]
### Description

$ loadbot polygon-edge --chain-id 6969
Error: invalid value
Uses:
loadbot [flags]

i try to change chain-id, jsonrpc but error when i type. position is running bootnode

![image](https://user-images.githubusercontent.com/93043826/192732111-d1cbae94-26d6-4fd6-a4ce-cbc1da640cf4.png)

**Comment by ivanbozic21** [comment, marked as answer]
Hi @avenbreaks,

Please check our official documentation about using loadbot command and example of how you can correctly start the loadbot on your end:

https://wiki.polygon.technology/docs/edge/additional-features/stress-testing/#start-the-loadbot

We will move this topic to discussion section, and let us know if you will need any additional help.

**Comment by avenbreaks** [comment]
thankyou!

**Comment by amandatsalis** [comment]
same error im trying to $ polygon-edge loadbot  --jsonrpc http://0.0.0.1:10002 --grpc-address 0.0.0.1:10000 --sender 0x6BC75033890cB64931d60c0438d05b05C1E01cFF --count 2000 --value 0x10 --tps 100

but notif show me : 

**an error occurred while running the loadbot: failed to extract sender account: failed to extract ECDSA private key from bytes: invalid key length (0B), should be 32B**

**Comment by JekaMas** [comment]
Do we have tests for API?

**Reply by amandatsalis** [reply]
> Do we have tests for API?

yes i have also tried jsonrpc and grpc api they run perfectly

**Reply by JekaMas** [reply]
Shall we have fuzzing and/or proprietary based testing on API as well? It might help if we mostly cover positive scenarios in tests at the moment.

**Comment by avenbreaks** [comment]
@JekaMas when im try to deploy new genesis.json on cloud notif showing me error : 
**failed to create libp2p stack: failed to listen on any addresses: [listen tcp4 20.xx.xx.xxx:1478: bind: cannot assign requested address]**

**Comment by avenbreaks** [comment]
sorry to ask again when i try to run node :
$ polygon-edge server --data-dir ./node-1 --chain genesis.json --libp2p 0.0.0.0:10001 --jsonrpc 0.0.0.0:10002 --grpc-address 0.0.0.0:10000 --nat 20.168.241.234 --block-time 7 --price-limit 1000000000 --prometheus 20.168.241.234:2311 --seal

all went well but when i try to run api command jsonrpc all my ip is not read the port is active too. but i try to change ip address 0.0.0.0 to ip vm also no solution
